pkg/network/render.go

```diff
@@ -16,7 +17,15 @@ func Render(conf *netv1.NetworkConfigSpec, manifestDir string) ([]*uns.Unstructu
 	objs := []*uns.Unstructured{}
 
 	// render default network
-	o, err := RenderDefaultNetwork(conf, manifestDir)
+	o, pluginName, err := RenderDefaultNetwork(conf, manifestDir)
+	if err != nil {
+		return nil, err
+	}
+	objs = append(objs, o...)
+
+	// render Multus
+	var crdRendered bool
+	o, crdRendered, err = RenderMultus(conf, manifestDir, pluginName)
```

Comment:
what about just passing ``conf`` to ``RenderMultus``, the pluginName can be derived from ``conf.DefaultNetwork.NetworkType``

Reply:
@zshi-redhat I'd rather keep the logic about what the CNI plugin name is alongside the plugin-specific logic